src/ios/CDVStatusBar.m

```diff
@@ -199,6 +199,14 @@ - (void) _ready:(CDVInvokedUrlCommand*)command
     }
 }
 
+- (void) height:(CDVInvokedUrlCommand*)command
+{
+    double statusBarHeight = [UIApplication sharedApplication].statusBarFrame.size.height;
+    CDVPluginResult* result = [CDVPluginResult resultWithStatus:CDVCommandStatus_OK messageAsDouble:statusBarHeight];
+    [result setKeepCallbackAsBool:YES];
```

Comment:
Just realized my [comment](https://github.com/apache/cordova-plugin-statusbar/pull/134#discussion_r514301773) might be misleading,`setKeepCallbackAsBool` should **not** be set to `YES`,
because this could causes a memory leak because the callback handler is never cleaned up.
```suggestion
```